report_py3o_fusion_server/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Py3o Report Engine - Fusion server support',
+    'summary': 'Let the fusion server handle format conversion.',
+    'version': '10.0.1.0.0',
+    'category': 'Reporting',
+    'license': 'AGPL-3',
+    'author': 'XCG Consulting,'
+              'ACSONE SA/NV,'
+              'Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/reporting-engine',
+    'depends': ['report_py3o'],
```

Comment:
@hbrunn The external dependencies are missing
```python
     'external_dependencies': {
         'python': ['py3o.template',
                    'py3o.formats']
},
```

Reply:
that's already taken care of by the base module this one depends on: https://github.com/OCA/reporting-engine/pull/180/files#diff-e165a51d925b6f5fdbab80a9482fd3fdR17

Reply:
@hbrunn Since you explicitly import these modules in `report_py3o_fusion_server`you must depend on these. It's never safe to rely on transitively acquired python dependencies.

Reply:
@lmignon I disagree, protecting the imports is enough imho, but I won't press the point. Now with external dependencies.